lib/ts/recipeImplementation.ts

```diff
@@ -199,7 +199,8 @@ export default function RecipeImplementation(recipeImplInput: {
                 if (!validationRes.isValid) {
                     errors.push({
                         validatorId: validator.id,
-                        reason: validationRes.reason
+                        reason: validationRes.reason,
+                        redirectPath: validationRes.redirectPath
```

Comment:
Im not sure I quite understand this - where does `redirectPath` come from? If it comes from the backend, how would the backend know what the path is?

Reply:
Please figure out a way to remove redirectToPath from website and web-js and adding that specific field in auth-react. Some ideas we discussed:
- Adding a callback to the auth-react's `validateGlobalClaimsAndRedirect` which will be called when claim validations fail.
- Making the error type generic here.